data/json/recipes/armor/suit.json

```diff
@@ -1700,6 +1700,23 @@
     "using": [ [ "sewing_standard", 28 ], [ "tailoring_kevlar_fabric", 16 ], [ "fastener_small", 2 ] ],
     "components": [ [ [ "sheet_kevlar_layered", 25 ] ] ]
   },
+  {
+    "result": "drying_linothorax",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_SUIT",
+    "skill_used": "tailor",
+    "difficulty": 3,
+    "skills_required": [ [ "fabrication", 3 ], [ "chemistry", 1 ] ],
+    "time": "4 h",
+    "autolearn": true,
```

Comment:
```suggestion
```
> In little more than two decades, Alexander the Great of Macedon (356-323 B.C.) conquered by military force nearly the entirety of the known world. Despite the fact that he led one of the most successful armies of all time, **surprisingly little is understood** about the main type of body armor that both Alexander and many of his men apparently wore.

This is not something you'd be able to just figure out.

Reply:
I feel like the general concept of laminated linen armor might be possible to intuit (not 100% sure on that), but either way the specific garment probably wouldn't be; I'll change it to book-only

Reply:
Now that I think about it, it should probably be removed from the clothing designer's portfolio